sass/patternfly-react/_notificationdrawer.scss

```diff
@@ -0,0 +1,45 @@
+.drawer-pf.drawer-alt-pf {
+  overflow-y: hidden;
+  display: flex;
+  flex-direction: column;
+
+  .drawer-pf-title {
+    position: relative;
+  }
+
+  .panel-group {
+    display: flex;
+    flex-direction: column;
+    position: initial;
+    bottom: initial;
+    top: initial;
+    overflow-y: auto;
+
+    .panel.panel-default.expanded {
+      min-height: 175px;
+      flex: 1 1 auto;
+      display: flex;
+      flex-direction: column;
+    }
+
+    .panel-collapse.in {
+      display: flex;
+      flex-direction: column;
+      flex: 1 1 auto;
+
+      .panel-body {
+        flex: 1 1 auto;
+        overflow-y: auto;
+      }
+    }
+  }
+
+  .drawer-pf-action {
+    padding: 10px 0;
+  }
+
+  .drawer-pf-action-link {
+    align-self: center;
+    margin: 0;
+  }
```

Comment:
I would replace all of this:
```
 .drawer-pf-action {
    padding: 10px 0;
  }

  .drawer-pf-action-link {
    align-self: center;
    margin: 0;
  }
```
with this:
```
  .drawer-pf-action {
    flex: none;
  }
```
We don't need this item to be a flex item. So setting `flex: none;` let's us use the spacing as it's defined in core without having to recreate it a different way.